src/main/java/com/infoshareacademy/controller/FoodPreferencesController.java

```diff
@@ -1,4 +1,68 @@
 package com.infoshareacademy.controller;
 
+
+import com.infoshareacademy.entity.food_preferences.FoodPreferences;
+import com.infoshareacademy.service.FoodPreferencesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.io.IOException;
+
+
+@Controller
 public class FoodPreferencesController {
-}
+
+    @Autowired
+    private FoodPreferencesService foodPreferencesService;
```

Comment:
```suggestion
    private final FoodPreferencesService foodPreferencesService;
```

zmienne które wstrzykujecie najlepiej jak są finalne